lib/http_clients/creditas/loan_api.ex

```diff
@@ -36,6 +36,17 @@ defmodule HttpClients.Creditas.LoanApi do
     end
   end
 
+  @spec cancel(Tesla.Client.t(), String.t()) :: {:error, any} | {:ok, String.t()}
+  def cancel(client, loan_id) do
+    body = %{"reason" => "CONTRACT_CANCELLATION"}
+
+    case Tesla.post(client, "/loans/#{loan_id}/cancel", body) do
+      {:ok, %Tesla.Env{status: 200}} -> {:ok, "OK"}
```

Comment:
Aqui não caberia retornar um `:no_content`?

Reply:
acho que fica melhor retornar só `:ok` nesse caso
```suggestion
      {:ok, %Tesla.Env{status: 200}} -> :ok
```

Reply:
o legal de retornar com o `:no_content` aqui seria manter o padrão de retornar tuplas .. acham q faz sentido?
{:ok, :no_content} no caso ..
mas pra mim tanto faz, ambas funcionam ok

Reply:
Temos outros casos que retornamos apenas o `:ok`, acho que um deles seriam ações de deletar que não retornam nada, então vejo que nesse caso a sugestão do `:ok` é melhor.

Reply:
eu acho que n tem pq manter tupla como retorno se a gente nao usar o valor da tupla 🤔
o contrato ficaria `:: :ok | {:error, any()}`
se alguem tiver um caso em mente que parece útil vamos discutir :)

Reply:
boa, parece correto

Reply:
alterei lá